src/pages/ListUsers.test.tsx

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Is this meant to be left in? If so what for?

Reply:
Good spot, I'll remove it